### ISSUE TITLE
[Feature Request] Feature to slow down belts

### ISSUE THREAD

**Issue by smetlov** [tobspr/shapez.io#630]
When at high belts speed it's hard to build new facilities. Items move too fast making it hard to recognize them. And you can't stop using Trash containers, because if you do so some branches stop producing items until other branches are free:
![image](https://user-images.githubusercontent.com/6358081/92004617-1b071300-ed4b-11ea-98ed-04f897c89e20.png)

So you can't see the output of the left branch until you place a Trash element on the right side.

So I want either to:
1. have an ability to select a zone and slow down belts speed to the acceptable value so I can recognize items or
2. have a 1-cell element that I can place at the beginning of the belt so make items move slower or something like that or
3. slow down the game speed globally.

**Comment by santa-klaus** [comment]
I agree that this would be useful. Two further possibilities to fix this problem:

4. Have a belt reader variety showing the shape, not throughput.
5. An option to freeze the game globally.

A workaround for those who don't want to wait for a feature: If you activate the option that the shapes on the belts are only rendered when you hover over them (don't remember the exact name, it's under Performance settings) you will have a small symbol on the belt showing you the shape (similar to my alternative 4).

**Comment by dengr1065** [comment]
There's no point in belt reader if it just shows the shape. Bind display to belt reader and here you go, but why? Just one belt is enough.

**Comment by jhamon** [comment]
I agree this would be useful. Once I reached belt speeds of 12 items/s I can't really identify objects anymore.
